.vscode/settings.json

```diff
@@ -1,13 +1,8 @@
-// Place your settings in this file to overwrite default and user settings.
 {
-  "editor.rulers": [
-      120
-  ],
+  "editor.rulers": [120],
   "editor.tabSize": 2,
-  "javascript.validate.enable": false,
-  "flow.useNPMPackagedFlow": true,
+  "javascript.validate.enable": true,
```

Comment:
Removes the flow rules, and enables the JavaScript validation - this turns on the inline TS warnings

Reply:
😎